src/org/omegat/core/search/SearchMatch.java

```diff
@@ -32,10 +32,12 @@
  */
 public class SearchMatch implements Comparable<SearchMatch> {
     private int start, end;
+    private String replacement;
 
-    public SearchMatch(int start, int end) {
+    public SearchMatch(int start, int end, String replacement) {
```

Comment:
I think we could have two constructors here, rather than forcing non-replacement instances to specify `null` for `replacement`.